src/contrib.md

```diff
@@ -44,6 +44,10 @@ And here is the formula in the separate block:
 
 $$E = mc^{2}$$
 
+###Some conventions
+
+* We have agreed as of issue [#83](https://github.com/e-maxx-eng/e-maxx-eng/issues/83) to express binomial coefficients with `\binom{n}{k}` or `{n \choose k}` instead of `C_n^k`.
```

Comment:
I'd add matching formulas to different expressions, so that it's evident that the difference is in the result of the rendering, not just in formula syntax.